platforms.c.in

```diff
@@ -0,0 +1,7 @@
+/* Autogenerated file, do not edit */
+#include <stdlib.h>
+
+@PLATFORM_DEFS@
+const struct debugcc_platform *platforms[] = {
+@PLATFORM_ARRAY@	NULL,
```

Comment:
Could you please add a newline before NULL?

Reply:
Was just trying out whether to use `'\n'.join()` which intersperses (first push), then switched to `+= '...\n'` which places it at the last entry and needs such hacks.

Not that it really matters, no-one should care about reading `build/platform.c.in` so we could have an empty line either way.